src/main/java/nwoolcan/model/utils/UnitOfMeasure.java

```diff
@@ -7,37 +7,5 @@
     /**
      * Unit of measures that can be used as quantity and parameter.
      */
-    Kilogram, Liter, Pound,
-    /**
-     * Unit of measures that cannot be used as quantity.
-     */
-    Celsius(false, true);
-
-    private boolean canBeQuantity;
-    private boolean canBeParameter;
-
-    UnitOfMeasure() {
-        this(true, true);
-    }
-
-    UnitOfMeasure(final boolean canBeQuantity, final boolean canBeParameter) {
-        this.canBeQuantity = canBeQuantity;
-        this.canBeParameter = canBeParameter;
-    }
-
-    /**
-     * Returns true if the unit of measure can be used as a quantity, false instead.
-     * @return true if the unit of measure can be used as a quantity, false instead.
-     */
-    public boolean canBeQuantity() {
-        return this.canBeQuantity;
-    }
-
-    /**
-     * Returns true if the unit of measure can be used as a parameter, false instead.
-     * @return true if the unit of measure can be used as a parameter, false instead.
-     */
-    public boolean canBeParameter() {
-        return this.canBeParameter;
-    }
+    Kilogram, Liter, Pound, Celsius;
```

Comment:
Why pound? that is not standard, not a problem anyway but we did decide to use only standard uom

Reply:
Can be removed